app/views/courses/TeacherClassView.coffee

```diff
@@ -294,14 +300,16 @@ module.exports = class TeacherClassView extends RootView
     user = @students.get(userID)
     selectedUsers = new Users([user])
     @enrollStudents(selectedUsers)
-  
+    window.tracker?.trackEvent $(e.currentTarget).data('event-action'), category: 'Teachers', classroomID: @classroom.id, userID: userID, ['Mixpanel']
+
   onClickBulkEnroll: ->
     courseID = @$('.bulk-course-select').val()
     courseInstance = @courseInstances.findWhere({ courseID, classroomID: @classroom.id })
```

Comment:
Whoops, looks like courseInstance and courseID aren't actually used here. They should be removed and not passed into the tracker event. Guess I mistakenly copied them over from the bulk-assign logic.